lms/djangoapps/branding_stanford/migrations/0002_auto_20190925_1058.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-09-25 17:58
+from __future__ import unicode_literals
+
+from django.db import migrations
+import opaque_keys.edx.django.models
```

Comment:
`from opaque_keys.edx.django.models import CourseKeyField` is what's used everywhere else

Reply:
Hrmmmm, looking at the logs of some of those migration files, it looks like they modified existing migrations,
hence why the import looks different than ours (which was manually generated by django).

Will that suffice here?

Reply:
@kluo It's saved to the DB as a string, so I'd assume it should be fine to follow edx' lead and not introduce a new migration, but instead modify the existing one. Thoughts?

https://github.com/edx/opaque-keys/blob/master/opaque_keys/edx/django/models.py#L198

Reply:
@kluo Oh, I also just noticed that you mentioned `CourseKeyField`, whereas this (and `models.py`) use `UsageKeyField`. Are you suggesting the model be updated too, or?

Reply:
yea I guess I meant we should also update the https://github.com/Stanford-Online/edx-platform/blob/open-release/hawthorn.master/lms/djangoapps/branding_stanford/models.py#L18
to `CourseKeyField` and just update the existing `0001_initial` migration like you mentioned

Reply:
Re:`models.py`

I'm trying to remember why I did that (6 months ago! 9cab273d748ef25beeeea4ca4f2bcef1b649dd7a)

It was in response to a deprecation warning, but why would I have used that instead of `CourseKeyField`?!
I want to say it was suggested by the deprecation warning???

Reply:
Yea I saw that warning, but it seems like it should be saying `Please use opaque_keys.edx.django.models.CourseKeyField instead` unless I'm missing something

Reply:
@kluo Done and squashed.